src/OpenTK.Mathematics/Vector/Vector4h.cs

```diff
@@ -129,7 +129,7 @@ public Vector4h(float x, float y, float z, float w, bool throwOnError)
         /// <summary>
         /// Initializes a new instance of the <see cref="Vector4h"/> struct.
         /// </summary>
-        /// <param name="v">The <see cref="Vector4"/> to convert.</param>
```

Comment:
Why this change? This constructor explicitly converts a normal vector.

Reply:
good quetion. I can't remember to have done this probably happened while merging...